### PR TITLE
CNF-16538: improve test coverage for alarms - repository 

### PR DESCRIPTION
**Pull request by pixelsoccupied**
this takes repo to about 89%

### CODE REVIEW

**Comment by pixelsoccupied** [comment]
flaky tests...investigating what's going on

**Comment by pixelsoccupied** [comment]
/hold

adding one more test in repo due improved edge case handling

File: internal/service/alarms/internal/db/repo/alarms_repository_test.go [hunk 1]
```diff
@@ -137,4 +138,720 @@ var _ = Describe("AlarmsRepository", func() {
 			})
 		})
 	})
+
+	Describe("DeleteAlarmDefinitionsNotIn", func() {
+		It("deletes alarm definitions not in provided IDs with correct object type ID", func() {
+			ids := []any{uuid.New(), uuid.New()}
+			objectTypeID := uuid.New()
+
+			mock.ExpectExec("DELETE FROM alarm_definition WHERE").
```
Comment (mlguerrero12) [inline internal/service/alarms/internal/db/repo/alarms_repository_test.go:147]: please use the TableName method from the db data model

Comment (pixelsoccupied) [reply internal/service/alarms/internal/db/repo/alarms_repository_test.go]: updated! we need a proper way to assert that the name of the table is correct. wonder if there's a way to do that for model package
